packages/config/config/devices/0x0190/adc-t_3000.json

```diff
@@ -0,0 +1,801 @@
+			"label": "HVAC Modes Enabled",
+			"description": "Which heating/cooling modes are available.",
+			"valueSize": 1,
+			"minValue": 3,
+			"maxValue": 31,
+			"defaultValue": 15,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off, Heat",
+					"value": 3
+				},
+				{
+					"label": "Off, Cool",
+					"value": 5
+				},
+				{
+					"label": "Off, Heat, Cool",
+					"value": 7
+				},
+				{
+					"label": "Off, Heat, Cool, Auto",
+					"value": 15
+				},
+				{
+					"label": "Off, Heat, Emergency Heat",
+					"value": 19
+				},
+				{
+					"label": "Off, Heat, Cool, Emergency Heat",
+					"value": 23
+				},
+				{
+					"label": "Off, Heat, Cool, Auto, Emergency Heat",
+					"value": 31
+				}
+			]
+		},
+		{
+			"#": "25",
+			"label": "Configurable Terminal Setting",
+			"description": "Changes control of configurable terminal",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 4,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "None",
+					"value": 0
+				},
+				{
+					"label": "W3, 3rd Stage Auxiliary Heat",
+					"value": 1
+				},
+				{
+					"label": "H, Humidifier",
+					"value": 2
+				},
+				{
+					"label": "DH, Dehumidifier",
+					"value": 3
+				},
+				{
+					"label": "External Air Baffle or Vent",
+					"value": 4
+				}
+			]
+		},
+		{
+			"#": "26",
+			"label": "Power Source",
+			"description": "Which source of power is utilized.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"readOnly": true,
+			"options": [
+				{
+					"label": "Battery",
+					"value": 0
+				},
+				{
+					"label": "C-Wire",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "27",
+			"label": "Battery Alert Threshold Low",
+			"description": "Battery Alert Range",
+			"unit": "%",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 30
+		},
+		{
+			"#": "28",
+			"label": "Battery Alert Threshold Very Low",
+			"description": "Very Low Battery Alert Range (percentage)",
+			"unit": "%",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 15
+		},
+		{
+			"#": "28",
+			"label": "Current Relay State",
+			"description": "Detected loads and current states of on-board relays",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 4294967295,
+			"unsigned": true,
+			"readOnly": true
+		},
+		{
+			"#": "30",
+			"label": "Remote Temperature Enable",
+			"description": "Enables remote temperature sensor instead of built-in.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 4,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "31",
+			"label": "Heat Differential",
+			"unit": "0.5°F",
+			"valueSize": 4,
+			"minValue": 2,
+			"maxValue": 20,
+			"defaultValue": 6
```

Comment:
I hope this is correct, the manual says this field is specified in 0.5 deg. F increments, so the default value is 3 degrees, specified here as 3 / 0.5 = 6

Reply:
seems correct.